dll/win32/browseui/shellfind/CFindFolder.cpp

```diff
@@ -235,6 +236,34 @@ static UINT SearchFile(LPCWSTR lpFilePath, _SearchData *pSearchData)
     return uMatches;
 }
 
+static BOOL FileNameMatch(LPCWSTR FindDataFileName, _SearchData *pSearchData)
+{
+    if (pSearchData->szFileName.IsEmpty() || PathMatchSpecW(FindDataFileName, pSearchData->szFileName))
```

Comment:
Since these are small functions now, you have the option of separating all these `||` constructs out into their own branches and having separate `return` statements. I don't have a strong opinion on whether that makes things more readable, but if you think it does, go ahead.

Reply:
I left it as it was for now.